CrazyLibrarians: Add initial version

gh215 opened the pull request.

gh215 committed:
CrazyLibrarians: Add initial version

amavrin commented on chapter-6/CrazyLibrarians/CrazyLibrarians/czlib.h:
```diff
@@ -0,0 +1,64 @@
+		cin >> text;
+		if (text.empty())
+		{
+			cout << "��������� �������� ������. ����������, ��������� �������.\n";
```
Давай что-нибудь придумаем, чтобы текст отображался корректно?
Если не получится - давай писать только по-английски

amavrin commented on chapter-6/CrazyLibrarians/CrazyLibrarians/czlib.h:
```diff
@@ -0,0 +1,64 @@
+	cout << name << ". ";
+	cout << "������ ��� �������? ����� ��������� � ";
+	cout << verb;
+	cout << ". \n";
```
Вывод функции выглядит так, как будто ты не проверял, что реально пишется.
Лишние или недостающие пробелы, неправильно расставлены заглавные буквы, некорректные знаки препинания, орфрграфические ошибки:
```
Известный путешественник Джон почти отказался от поисков
 Затерянного города коробки , когда в один прекрасный день коробки нашел исследователя.
Окруженный 7 коробки. а слеза попала в Джон's рука.
После стольких лет поиски наконец-то закончились. И тогда. коробки
быстро сожралиДжон. Мораль сей истории? Будте осторожны в делать.
```
Это выглядит неаккуратно

amavrin reviewed (changes requested):
В целом, программа рабочая.

Но хочу отметить, что, как правило (хотя и не всегда), определение функций размещают в файлах .cpp, а в .h - объявления.

Предлагаю переделать этот момент, а также
* исправить форматирование
* решить проблемы с русским текстом

gh215 committed:
CrazyLibrarians: temp.

gh215 committed:
CrazyLibrarians: Russian has been replaced by English, two new character validation functions have been added,
one for askText, which checks that the characters entered by the user are only letters,
and one for askNumber, which checks that the characters entered by the user are only numbers.

amavrin reviewed (approved):
Отлично

Только расскажи потом, что было требуемым улучшением программы, и почему это улучшение
